### ISSUE TITLE
Chinese character alignment issue

### ISSUE THREAD

**Issue by lybur** [SimformSolutionsPvtLtd/flutter_showcaseview#254]
**Describe the bug**
The chinese character for showcaseview will become vertical, but this issue only occurs when I release the android app in play console, it's working fine during debug. Does someone has an idea of what's going wrong? 

**To Reproduce**
Unable to replicate the issue as debug and release mode in Flutter is working fine.

**Screenshots**
**Expected**
<img src="https://user-images.githubusercontent.com/110459982/182348973-5ff0b3e7-e333-4627-a1bc-c195a3f7db89.PNG" width="300">
**After release:**
<img src="https://user-images.githubusercontent.com/110459982/182349187-d61b72e7-f44b-4afe-9192-ed4f4c0ec8e5.JPEG" width="300">

Smartphone:
 - Device: All android devices

**Comment by ToZhanChi** [comment]
这个问题得到解决了吗？

**Comment by MehulKK** [comment]
@ToZhanChi  : No Its yet to resolve... once we fix this will update you on same.

Thanks

**Comment by ToZhanChi** [comment]
@MehulKK 我发现只在bottomNavigationBar和TabBar中出现这个对齐问题。其他地方没有问题

**Comment by HappyMakadiyaS** [comment]
Hey @lybur and @ToZhanChi 
We have released the new version of showcase view. Can you you check that your issue is resolved or not?

**Comment by HappyMakadiyaS** [comment]
And, did you use obfuscation when generating the apk?

**Comment by lybur** [comment]
> And, did you use obfuscation when generating the apk?

No we did not use obfuscation and we are using app bundle.
We actually manage to solve this issue before you released the new version, so I'm not sure if this fix is legit, maybe need @ToZhanChi to confirm again.

How we able to solve is that we found out there's some difference in Android SDK and gradle version among my members, so we just tally all the version and it worked out.
